### DISCUSSION TITLE
ProScan & SDR Sharp

### DISCUSSION THREAD

**Discussion by ryanhowell18** [chuot/rdio-scanner#410] [General]
Hello everyone,

I'm currently facing a challenge with my rdio-scanner setup, involving multiple scanners and Airspy R2 devices. Despite my efforts, I've encountered difficulties in retrieving metadata from these scanners. I've attempted to follow instructions related to Proscan, but unfortunately, I haven't achieved success after several tries.

For the Airspy R2's, I utilize SDR Sharp software. Specifically, my goal is to display channel names and frequencies, yet my attempts have proven unsuccessful so far. I'm reaching out to seek suggestions and guidance on how to retrieve this metadata. If anyone has insights or recommendations, I would greatly appreciate your help

Thank you in advance for your support.

**Comment by Thinline-Dynamic-Solutions** [comment]
Have you tried SDR Trunk?

*Michael Chambers - Ohio Radio Scanner Network*
Founder/Manager/Support
***@***.**** ***@***.***>

**Reply by ryanhowell18** [reply]
Hello, As in the software or the dirwatch option?
